packages/worker/src/worker-options.ts

```diff
@@ -431,3 +523,25 @@ export function compileWorkerOptions(opts: WorkerOptionsWithDefaults): CompiledW
     loadedDataConverter: loadDataConverter(opts.dataConverter),
   };
 }
+
+function reportWorkerOptions(options: WorkerOptionsWithDefaults) {
+  Runtime.instance().logger.info(`Worker options:`);
+  Runtime.instance().logger.info(
+    JSON.stringify(
```

Comment:
Hmm.. why JSON.stringify instead of letting the logger take care of that?

Reply:
Could replace these 2 lines with: `logger.info('Worker created with options', options)` and move this to `Worker.create`